packages/core/src/core/context/request-context.ts

```
@@ -81,9 +112,19 @@ export class RequestContext {
 	 * @returns The current RequestContext instance.
 	 */
 	static currentRequestContext(): RequestContext {
-		if (RequestContext?.logging) console.log('RequestContext: getting context ...');
-		const context = RequestContext?.clsService?.get(RequestContext.name);
-		if (RequestContext?.logging) console.log('RequestContext: got context with Id:', context?._id);
+		// Log if logging is enabled
+		if (this.loggingEnabled) {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)